typescript/introduction/intro-typescript/what-is-typescript.md

```diff
@@ -0,0 +1,47 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Intro to TypeScript
+
+---
+## Content
+
+If you're old enough, you might remember when JavaScript was just a web development language. Due to its versatility, JavaScript is now used in many applications including neural network design, power backends and many more.
+
+You're probably thinking "What's JavaScript got to do with it -- isn't this a TypeScript course?". Let us explain!
+
+Because of the high adoption rate of JavaScript, many tools (such as Angular, React, Node.js, etc) were created to allow programmers to write better/safer JavaScript code. 
```

Comment:
I wouldn't say `Angular, React, Node` allowed `better/safer` code to be written; It's rather they allowed more specialized code to be written, that adhere to a certain programming style (in case of React/Angular) or allow JavaScript to be run on any computer, not just in browsers

Reply:
Better examples would be Dart, Flow, Kotlin JS (Which is BAE btw)

Reply:
Something like this @mihaiberq @sourabhv ?

```suggestion
Because of the high adoption rate of JavaScript, many tools were created to allow programmers to write code on multiple platforms, specialized for various programming styles. 
```

Reply:
What does *write code on multiple platforms*? And *specialized for various programming styles*? Maybe explain these in insight briefly?

Reply:
How about this? cc @sourabhv 

```suggestion
Because of the popularity of JavaScript, many tools were created to further support different programming patterns or environments. Some examples are tools that aim to improve web app development [such as Angular](https://angular.io/), enable mobile app development (such as [React Native](https://reactnative.dev/), or enable building backend servers (such as [Node.js](https://nodejs.dev/)).
```

Reply:
Looks good, but since this is TS, an extension/transpiler of JS not implementation, we should also give examples of transpilers, not frameworks built on top of JS

Reply:
Thats why I mentioned Dart Kotlin JS and Flow, More would be Coffeescript, Babel,